botbuilder-instumentation/Managers/SentimentManager.cs

```diff
@@ -63,18 +62,14 @@ public class SentimentManager : ISentimentManager
 
         private async Task<BatchResult> GetSentiment(string apiKey, string jsonSentimentInput)
         {
-            using (var client = new HttpClient())
-            {
-                client.BaseAddress = new Uri(_cognitiveServiceApiEndpoint);
+            const string sentimentRoute = "text/analytics/v2.0/sentiment";
+            var headers = new Dictionary<string, string> {{"Ocp-Apim-Subscription-Key", apiKey}};
```

Comment:
If this method is being called a lot, it would make more sense to make the dictionary const outside of the method in order to create this object only once.